use-timescale/hypertables/create.md

```diff
@@ -24,10 +24,8 @@ available](/api/:currentVersion:/hypertable/create_hypertable_old/).
 To create a hypertable, you need to create a standard PostgreSQL table, and then
 convert it into a hypertable.
 
-Hypertables are intended for time-series data, so your table needs a column that
-holds time values. This can be a timestamptz, date, or integer. Ensure that you
-set the datatype for the `time` column as `timestamptz` and not `timestamp`. For
-more information, see [PostgreSQL timestamp][postgresql-timestamp].
+Hypertables are designed for real-time analytics and partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, a best practice is to 
+use `timestamptz` instead. For why the use of `timestamp` is discouraged, see [PostgreSQL timestamp][postgresql-timestamp].
```

Comment:
It is possible to use integers as well for partitioning, even though they do not represent a time. This is not normally what we recommend though.

```suggestion
Hypertables are designed for real-time analytics and typically partitioned by columns that hold time values. These can be of the `timestamptz`, `date`, or `integer` types. While `timestamp` is also supported, a best practice is to 
use `timestamptz` instead. For why the use of `timestamp` is discouraged, see [PostgreSQL timestamp][postgresql-timestamp].
```